src/util/PolarUtils.ts

```diff
@@ -29,6 +29,17 @@ export const getMaxRadius = (
     Math.abs(height - (offset.top || 0) - (offset.bottom || 0)),
   ) / 2;
 
+type FormatAxisMapProps = {
```

Comment:
I think instead of defining this newly, we could reuse the union of PolarAxisProps and CartesianAxisProps.